src/usr/local/emhttp/plugins/open.files/data.php

```
@@ -0,0 +1,41 @@
+    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+require_once dirname(__FILE__) . "/include/common.php";
+
+$prefix = "/plugins/open.files/data.php";
+
+$app = AppFactory::create();
+$app->addRoutingMiddleware();
+$errorMiddleware = $app->addErrorMiddleware(true, true, true);
+
+$app->get("{$prefix}/files", function (Request $request, Response $response, $args) {
```

PHPStan (2.1.15)

Parameter $request of anonymous function has invalid type Psr\Http\Message\ServerRequestInterface.


(class.notFound)